README.md

```diff
@@ -9,31 +9,32 @@ expectations. It is applicable to GDI projects.
 > handled upstream. For information on OpenTelemetry, see the [OpenTelemetry
 > Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md)
 
-The following components are currently in scope:
+The following specification sections are currently in scope:
 
 - [Configuration](specification/configuration.md)
 - [Repository](specification/repository.md)
 - [Versioning](specification/versioning.md)
 
-GDI projects MUST adopt GDI specification changes by their next minor release
+GDI projects MUST adopt GDI specification changes by their next `MINOR` release
 and within three months (whichever is sooner). The GDI specification and GDI
-projects MUST remove any deprecated specification at the next major release.
+projects MUST remove any deprecated specification at the next `MAJOR` release.
```

Comment:
I'd frame this as `MUST NOT REMOVE before next MAJOR release` and allow keeping it even more multiple major releases.

Reply:
I think this is a valid suggestion, but I don't think it is related to the changes being introduced here. Resolving this and tracking in #101 .